doc/services/device_mgmt/smp_groups/smp_group_0.rst

```diff
@@ -840,3 +842,197 @@ where:
+    |                  | a map, that is dependent on bootloader backednd and query.              |
+    +------------------+-------------------------------------------------------------------------+
+    | ...              | Parameter characteristic information.                                   |
+    +------------------+-------------------------------------------------------------------------+
+
+Parameter may be accompanied by additional, parameter specific, information keywords with
+assigned values.
+
+In case of error the CBOR data takes the form:
+
+.. tabs::
+
+   .. group-tab:: SMP version 2
+
+      .. code-block:: none
+
+          {
+              (str)"err" : {
+                  (str)"group"    : (uint)
+                  (str)"rc"       : (uint)
+              }
+          }
+
+   .. group-tab:: SMP version 1 (and non-group SMP version 2)
+
+      .. code-block:: none
+
+          {
+              (str)"rc"       : (int)
+          }
+
+where:
+
+.. table::
+    :align: center
+
+    +------------------+-------------------------------------------------------------------------+
+    | "err" -> "group" | :c:enum:`mcumgr_group_t` group of the group-based error code. Only      |
+    |                  | appears if an error is returned when using SMP version 2.               |
+    +------------------+-------------------------------------------------------------------------+
+    | "err" -> "rc"    | contains the index of the group-based error code. Only appears if       |
+    |                  | non-zero (error condition) when using SMP version 2.                    |
+    +------------------+-------------------------------------------------------------------------+
+    | "rc"             | :c:enum:`mcumgr_err_t` only appears if non-zero (error condition) when  |
+    |                  | using SMP version 1 or for SMP errors when using SMP version 2.         |
+    +------------------+-------------------------------------------------------------------------+
+
+Bootloader Information: MCUboot
+===============================
+
+In case when MCUboot is application bootloader empty request will
+be responded with:
+
+.. code-block:: none
+
+    {
+        (str)"bootloader"      : (str)"MCUboot"
+    }
+
+Currently "MCUboot" supports querying for mode of operation:
+
+.. code-block:: none
+
+    {
+        (str)"query"           : (str)"mode"
+    }
+
+Response to "mode" is:
+
+.. code-block:: none
+
+    {
+        (str)"mode"                     : (int)
+        (str,opt)"no-downgrade"         : (bool)
+    }
+
+where "mode" is one of:
+
+.. table::
+    :align: center
+
+    +-----+-----------------------------------------------------+
+    | -1  | Unknown mode of MCUboot.                            |
+    +-----+-----------------------------------------------------+
+    |  0  | MCUboot is in single application mode.              |
```

Comment:
Probably best to use these enum values for this: https://github.com/mcu-tools/mcuboot/blob/main/boot/bootutil/include/bootutil/boot_status.h#L118

Reply:
Done